HumanControls/DashPage.cpp

```diff
@@ -10,28 +10,50 @@ void DashPage::paint(DisplayController &display, bool isActivated, JsonElement &
 {
     display.clear();
 
-    if (object["hCtrl"]["conn"].asString()[0] == 'A')
+    if (canActivate() && isActivated)
```

Comment:
Is `canActivate()` redundant in this case?

Reply:
Done.